drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -151,6 +148,7 @@ def items_without_answers():
             for item in items:
                 del item['feedback']
                 del item['zone']
+                item['displayName'] = self.ugettext(item['displayName'])
```

Comment:
@asadiqbal08 See @bradenmacdonald's [comment](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/66#discussion_r57737584) -- you shouldn't call `self.ugettext` on user-provided strings. These include the string in this line (`item[displayName]`) and most of the strings below; calling it on `field.help` is fine since help texts for fields are defined in the code ([example](https://github.com/edx-solutions/xblock-drag-and-drop-v2/blob/asadiqbal08/WL-373/drag_and_drop_v2/drag_and_drop_v2.py#L37)) and can't be changed by course authors.